apps/nextra/components/landing/sections/BlockchainSection.tsx

```diff
@@ -60,47 +60,47 @@ export function BlockchainSection() {
         <FeatureCard
           label={t.randomnessLabel}
           description={t.randomnessDescription}
-          href="" // TODO: Add link to relevant docs page
+          href={`/${locale}/build/move/move-on-aptos/cryptography#verifying-randomness-from-the-drand-beacon`}
```

Comment:
No the page titles and routes should be the same. Otherwise the i18n routing won't work properly since it expects the page to live at that route, just under a different locale prefix

Reply:
I think it might be better to use relative paths here but locale absolute paths work as well

Reply:
I'm not worried about the filename paths.  Those should be consistent.  I meant the anchors e.g. `#verifying-the-randomness blah blah`. Those are autogenerated from headings, unless we do something special